Dug11 dasharo hsi 3

filipleple opened the pull request:
*ref: sha-1511*

filipleple committed:
pages/dug_11/0X_dasharo_path_to_hsi_3.md: add

Signed-off-by: Filip Lewiński <filip.lewinski@3mdeb.com>

SergiiDmytruk commented on pages/dug_11/0X_dasharo_path_to_hsi_3.md:
```diff
@@ -0,0 +1,268 @@
+## Why a metric like HSI matters
+
+* Platform security is a mess of low level knobs and vendor features
+* HSI compresses that into a single comparable level
```
```suggestion
* HSI compresses that into a single comparable integer that represents a security level
```

SergiiDmytruk commented on pages/dug_11/0X_dasharo_path_to_hsi_3.md:
```diff
@@ -0,0 +1,268 @@
+
+* From the OS, cbmem and tpm2 tools show BootGuard status and PCR0 details
+* Dasharo Setup UI exposes BootGuard state without parsing logs
+* fwupdmgr security or fwupdtool security give a high level HSI score backed by the same evidence
```
```suggestion
* `fwupdmgr security` or `fwupdtool security` give a high-level HSI score backed by the same evidence
```

pietrushnic commented:
@filipleple I'm good with @SergiiDmytruk's review. He wrote a blog post, which I reviewed, so unless there are some concerns about the business-related director, I would skip that review, because I have quite a lot of other stuff to do. Stick to the blog post, and it should be good IMHO.

filipleple committed:
pages/dug_11/0X_dasharo_path_to_hsi_3.md: address review

Signed-off-by: Filip Lewiński <filip.lewinski@3mdeb.com>

filipleple committed:
pages/dug_11/0X_dasharo_path_to_hsi_3.md: wrap at 80 chars

Signed-off-by: Filip Lewiński <filip.lewinski@3mdeb.com>

filipleple committed:
pages/dug_11/0X_dasharo_path_to_hsi_3.md: clarify around locality logging

Signed-off-by: Filip Lewiński <filip.lewinski@3mdeb.com>

filipleple committed:
pages/dug_11/0X_dasharo_path_to_hsi_3.md: cut down on the business related messaging

Signed-off-by: Filip Lewiński <filip.lewinski@3mdeb.com>

filipleple committed:
pages/dug_11/0X_dasharo_path_to_hsi_3.md: satisfy pre-commit

Signed-off-by: Filip Lewiński <filip.lewinski@3mdeb.com>

filipleple committed:
diagrams/coreboot-upstreaming.sh: satisfy pre-commit

Signed-off-by: Filip Lewiński <filip.lewinski@3mdeb.com>

filipleple committed:
pages/dug_11/0X_dasharo_path_to_hsi_3.md: fix typo

Signed-off-by: Filip Lewiński <filip.lewinski@3mdeb.com>

SergiiDmytruk committed:
pages/dug_11/0X_dasharo_path_to_hsi_3.md: {TME} -> (TME)

The first one doesn't even get displayed.

Signed-off-by: Sergii Dmytruk <sergii.dmytruk@3mdeb.com>

filipleple committed:
pages/dug_11/0X_dasharo_path_to_hsi_3.md: trim the speaker notes a bit

I had to remove a couple sentences that didn't add much to the picture,
because I wasn't making it in the ~15 minutes

Signed-off-by: Filip Lewiński <filip.lewinski@3mdeb.com>

filipleple committed:
pages/vpub_0x11/0X_dasharo_path_to_hsi_3.md: migrate from DUG 11

Signed-off-by: Filip Lewiński <filip.lewinski@3mdeb.com>

filipleple commented:
@SergiiDmytruk @BeataZdunczyk moved from `dug_11` (wrong) to `vpub_0x11` (following @mkopec)

also trimmed down a couple sentences, because I wasn't making it in 15 minutes

filipleple committed:
slides-template.md: change to DUG 12, vPub 0x11

Signed-off-by: Filip Lewiński <filip.lewinski@3mdeb.com>
